components/log-viewer-webui/client/package.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "log-viewer-webui-client",
+  "version": "0.1.0",
+  "description": "",
+  "main": "src/index.jsx",
+  "scripts": {
+    "build": "webpack --define-process-env-node-env production",
+    "lint:check": "npx eslint --no-eslintrc --config package.json src webpack.config.js",
+    "lint:fix": "npx eslint --fix --no-eslintrc --config package.json src webpack.config.js"，
```

Sorry the typo of this full-width comma.
```suggestion
    "lint:fix": "npx eslint --fix --no-eslintrc --config package.json src webpack.config.js",
```